secretsauce/apps/account/urls.py

```diff
@@ -18,4 +18,6 @@
     path('', include('djoser.urls.authtoken')),
     path('invitation/', views.InvitationCreator.as_view()),
     path('invitation/<uuid:pk>', views.InvitationDetail.as_view()),
+    path('company/', views.CompanyList.as_view()),
+    path('company/', views.CompanyDetail.as_view()),
```

Comment:
wrong path name? Cant access invidividual companies

Reply:
Changed in latest commit